src/Wrapper.js

```diff
@@ -61,15 +61,21 @@ class Wrapper extends React.Component {
     walletProviderId: '',
     walletWeb3: null,
   }
+
+  static getDerivedStateFromProps = ({ autoClosingPanel }) => {
```

Comment:
With these changes when you resize to "small" the `MenuPanel` stays opened, what about we close the `MenuPanel` in such case to give emphasis to the content:

```JavaScript
componentDidUpdate(prevProps) {
  const { autoClosingPangel } = this.props
  if (autoClosingPanel !== prevProps.autoClosingPanel) {
    this.setState({ menuPanelOpened: !autoClosingPanel})
  }
}
```